### ISSUE TITLE
Assigning Tasks

### ISSUE THREAD

**Issue by reedacartwright** [rachelss/SISRS#33]
With development of SISRS taking off this semester in two locations, it is important that we figure out who is addressing which issue or pull request. The easiest way to do it is if developers self-assign issues they are addressing and we have occasional skype meetings.

Thoughts @rachelss?

**Comment by anderspitman** [comment]
@rachelss, realated to this, @reedacartwright asked me to help out with a few SISRS issues if I can. Is there any specific priority of what you'd like me to look at? Preferably something I can work on using my local workstation.

**Comment by rachelss** [comment]
Yes we need to assign! I'll think about who might do what more substantially beyond issues.
